README.rst

```diff
@@ -486,6 +486,31 @@ The ZCML builder builds a ZCML file:
            .with_node('i18n:registerTranslations', directory='locales'))
 
 
+Portlet builder
+~~~~~~~~~~~~~~~
+
+The ``ftw.builder`` ships with a few builders for Plone portlets, but the
+idea is that you can easily craft your own builders for your portlets or
+extend existing builders.
+
+Example:
+
+.. code:: python
+
+    from ftw.builder.portlets import PlonePortletBuilder
```

Comment:
it might be helpful if you include the ``builder_registry`` import here, just for copy/paste :wink: